Source/WebKit/Shared/AuxiliaryProcess.cpp

```diff
@@ -132,8 +135,14 @@ void AuxiliaryProcess::initializeProcessName(const AuxiliaryProcessInitializatio
 {
 }
 
-void AuxiliaryProcess::initializeConnection(IPC::Connection*)
+void AuxiliaryProcess::initializeConnection(IPC::Connection* connection)
 {
+#if OS(LINUX) && USE(GLIB)
+    if (isInsideFlatpak())
```

Comment:
Two thoughts here:

1. We need to do this for bubblewrap sandbox as well, not only flatpak sandbox. I would at least add a comment here to explain why the code is correct, because it looks like it's not. I see that when we are the web process, this check will return true because the bubblewrap sandbox _looks_ like a Flatpak sandbox, i.e. `isInsideFlatpak` just checks for Flatpak in the UI process, but in the web process it additional checks for WebKit's own sandbox. Maybe we should follow up on this because it's quite confusing.
2. Sending credentials is not necessary if we are the parent process rather than the child process. But I suppose it's also harmless, and not worth adding code to avoid doing so?